### PR TITLE
Merchant dashboard

### PR DESCRIPTION
**Pull request by KelsiePorter**
User Stories 1 - 5

### CODE REVIEW

File: spec/features/merchants/show_spec.rb [hunk 1]
```diff
@@ -206,18 +206,26 @@
...
+      within("#unshipped_item_invoice_#{@invoice_4.id}") do
+        expect(page).to have_link("Invoice ID #{@invoice_4.id}", href: "/merchants/#{@merchant_1.id}/invoices/#{@invoice_4.id}")
         expect(page).to have_content(@item_1.name)
         expect(page).to_not have_content(@item_2.name)
       end
-      within("#unshipped_item_#{@item_2.id}") do
+      within("#unshipped_item_invoice_#{@invoice_18.id}") do
+        expect(page).to have_link("Invoice ID #{@invoice_18.id}", href: "/merchants/#{@merchant_1.id}/invoices/#{@invoice_18.id}")
+        expect(page).to have_content(@item_2.name)
+        expect(page).to_not have_content(@item_1.name)
```
Comment (anhtran811) [inline spec/features/merchants/show_spec.rb:219]: way to test your edge cases here!

File: app/models/merchant.rb [hunk 2]
```diff
@@ -6,7 +6,7 @@ class Merchant < ApplicationRecord
   has_many :transactions, through: :invoices, dependent: :destroy
 
   def unshipped_items 
-    items.select('items.name, invoice_items.invoice_id as invoice_id')
+    items.select('items.id, items.name, invoice_items.invoice_id as invoice_id')
```
Comment (anhtran811) [inline app/models/merchant.rb]: Great job recognizing the change that had to be made to get this query to work. That one was a beast!
